src/garage/envs/__init__.py

```diff
@@ -8,9 +8,10 @@
 from garage.envs.half_cheetah_vel_env import HalfCheetahVelEnv
 from garage.envs.normalized_env import normalize
 from garage.envs.point_env import PointEnv
+from garage.envs.rl2_env import RL2Env
```

Comment:
can this be part of the `garage.tf.algos.rl2` module instead?

is there any way that RL2 can automatically wrap the environment with this wrapper?

Reply:
Yes I will put this inside rl2 class and make RL2 automatically wrap the environment with this wrapper

Reply:
Actually, the env never get passed to RL2 -- It only lives in the runner so RL2 can't wrap it.

Reply:
ah. this will have to wait for the refactor where the algorithm owns the sampler. 

can you please file an issue about this?